### PR TITLE
[Build] travis osx

### PR DESCRIPTION
**Pull request by timosachsenberg**
enable osx builds on travis

### CODE REVIEW

File: tools/travis/osx-cibuild.cmake [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+set(CTEST_SOURCE_DIRECTORY "$ENV{SOURCE_DIRECTORY}")
+set(CTEST_BINARY_DIRECTORY "${CTEST_SOURCE_DIRECTORY}/_build")
+
+message(STATUS "CTEST_SOURCE_DIRECTORY: ${CTEST_SOURCE_DIRECTORY}")
+message(STATUS "CTEST_BINARY_DIRECTORY: ${CTEST_BINARY_DIRECTORY}")
+
+
+#/usr/local for homebrew
+set(INITIAL_CACHE
+"CMAKE_PREFIX_PATH=$ENV{SOURCE_DIRECTORY}/contrib\;/usr/local
```
Comment (cbielow) [inline tools/travis/osx-cibuild.cmake]: this seems to be the only difference between the osx vs lnx file.
So to reduce maintenance effort and code duplication, could this be solved via a variable in 'cibuild.sh'?

Comment (timosachsenberg) [inline tools/travis/osx-cibuild.cmake]: I think it is possible by setting an additional environment variable in cibuild.sh and using 
"CMAKE_PREFIX_PATH=$ENV{SOURCE_DIRECTORY}/contrib\;$ENV{OS_PREFIX_PATH}
or similar.
Any suggestions for a good name?

Comment (cbielow) [inline tools/travis/osx-cibuild.cmake]: Nice.
Os_prefix_path sounds good to me. 
-- 
Sent via mobile phone.

Am 2. September 2015 13:04:59 GMT+01:00, schrieb Timo Sachsenberg <notifications@github.com>:

**Comment by cbielow** [comment]
any idea why these are failing?

	352 - ConsensusIDAlgorithmBest_test (Failed)
	356 - ConsensusIDAlgorithmWorst_test (Failed)
	428 - SVMWrapper_test (Failed)
	1567 - TOPP_ConsensusID_5_out1 (Failed)

our nightly builds seem to be broken in other terms so its hard to compare...

**Comment by timosachsenberg** [comment]
I get some errors on mac but the other tests run fine:
http://cdash.openms.de/viewTest.php?onlyfailed&buildid=143217
Any ideas why these yield different results?

**Comment by timosachsenberg** [comment]
needs adaption to new container builds

**Comment by timosachsenberg** [comment]
@hendrikweisser any idea why this fails: http://cdash.openms.de/viewTest.php?onlyfailed&buildid=145418

**Comment by hendrikweisser** [comment]
@timosachsenberg:
> any idea why this fails: http://cdash.openms.de/viewTest.php?onlyfailed&buildid=145418

Numerical instability. There are search hits ordered by score, and for cases with theoretically identical, but numerically not-quite-identical scores, they end up in a different order than expected. So the tests have to be made a bit more forgiving.

**Comment by timosachsenberg** [comment]
I will merge as getting build information early is better but we should make sure to fix the tests soon

**Pull request merged**